docs/faq.md

```diff
@@ -122,5 +122,62 @@ user is in control. Windows and views created by the app are positioned by the
 OS shell. Its UX is either drawn using XAML or Composition objects or a
 framework that produces XAML or Composition objects.
 
-## I don't see my question here! [Create an issue to ask a question or start a discussion](https://github.com/microsoft/ProjectReunion/issues/new/choose).
+## How will I consume Project Reunion components? How will they be packaged?
+Project Reunion components will be available as NuGet packages and Framework packages. 
+Developers can add Nuget packages to their build environments and or reference shared code 
+that will be available as framework packages.
+
+At developer consumption time it's a Nuget package with metadata and the things your 
+languages need to build against it. 
+At runtime, it's a set of DLLs distributed using a framework package. Your app doesn't 
+need to redistribute the DLLs directly - your app would either depend on the framework 
+package specifically (in the case of a packaged app) or would deploy our upcoming 
+ProjectReunion.MSIX that contains the framework package as part of your app's unpackaged 
+installer (similar to how an app carried the VC Runtimes or the DirectX, etc.)
+
+## What is a Framework Package?
+Framework packages are ways for apps to share code from a common source, rather than 
+carrying it with themselves. Apps just need to reference the framework package in their 
+manifest.
```

Comment:
Markdown requires a blank line between paragraphs. I think you need one here after "manifest" and on line 131, etc.